Source/PeterKottas.DotNetCore.WindowsService.Example/ExampleService.cs

```diff
@@ -1,25 +1,25 @@
-﻿using PeterKottas.DotNetCore.WindowsService.Interfaces;
+﻿using Microsoft.Extensions.PlatformAbstractions;
+using PeterKottas.DotNetCore.WindowsService.Interfaces;
 using System;
 using System.IO;
-using System.Diagnostics;
-using Microsoft.Extensions.PlatformAbstractions;
-using System.ServiceProcess;
-using System.Threading.Tasks;
+using System.Timers;
 
 namespace PeterKottas.DotNetCore.WindowsService.Example
 {
-    public class ExampleService : IMicroService
+	public class ExampleService : IMicroService
     {
-        private IMicroServiceController controller;
+        private IMicroServiceController _controller;
```

Comment:
Could you pls rename this back? I know it's common naming convention but it's not really used anywhere else so it would be breaking what we have there atm.

Reply:
No problem